### PR TITLE
Overhaul ui of  profile modification settings

### PR DESCRIPTION
**Pull request by Jikstra**
# Before
Screenshot_20201121_214038

# After
Peek 2020-11-21 21-37

# ToDo
- [x] username is not getting updated in settings view but after a restart of the application?

### CODE REVIEW

File: src/main/deltachat/burnerAccounts.ts [hunk 1]
```diff
@@ -1,4 +1,3 @@
-import fetch from 'node-fetch'
```
Comment (Simon-Laux) [inline src/main/deltachat/burnerAccounts.ts:1]: nah that line belongs here, I guess you forget an `npm install` to install the typings...

File: src/renderer/components/dialogs/Settings-Profile.tsx [hunk 2]
```diff
@@ -0,0 +1,205 @@
...
+  const onClickRemovePicture = () => setProfilePicture('')
+
+  const codepoint = displayName && displayName.codePointAt(0)
+  const initial = codepoint
+    ? String.fromCodePoint(codepoint).toUpperCase()
+    : '#'
+
+  return (
+    <div className='profile-image-selector'>
+      {/* TODO: show anything else when there is no profile image, like the letter avatar */}
```
Comment (Simon-Laux) [inline src/renderer/components/dialogs/Settings-Profile.tsx:105]: remove this deprected todo comment

**Comment by Simon-Laux** [comment]
```
ipcMain.on('setConfig', (e, key, value) => {
    e.returnValue = dcController.settings.setConfig(key, value)
})
```
- [ ] remove the now un-referenced ipc function?

File: src/main/deltachat/burnerAccounts.ts (hunk 1 above)
Comment (Jikstra) [reply src/main/deltachat/burnerAccounts.ts]: Ouups :D

**Pull request merged**